template/src/ontology/run.sh.jinja2

```diff
@@ -63,7 +63,14 @@ if [ -n "$TAG_IN_IMAGE" ]; then
   ODK_IMAGE=$(echo $ODK_IMAGE | awk -F':' '{ print $1 }')
 fi
 ODK_TAG=${ODK_TAG:-latest}
-ODK_JAVA_OPTS={% if project.robot_java_args is defined and project.robot_java_args|length %}${ODK_JAVA_OPTS:-{{ project.robot_java_args }}}{% else %}${ODK_JAVA_OPTS:--Xmx8G}{% endif %}
+{% if project.robot_java_args is defined and project.robot_java_args|length %}
+ODK_JAVA_OPTS=${ODK_JAVA_OPTS:-{{ project.robot_java_args }}}
+{% else %}
+if [ -z "$USE_SINGULARITY" ]; then
+    {% raw %}DEFAULT_MAX_MEM=$(bc <<<"($(docker info --format={{.MemTotal}}) * .9) / (1024*1024*1024)")G{% endraw %}
+fi
+ODK_JAVA_OPTS=${ODK_JAVA_OPTS:--Xmx$DEFAULT_MAX_MEM}
```

Comment:
Where is this DEFAULT_MAX_MEM coming from?

Reply:
Oops. It should have been set to the old default (`8G`) in the case where `USE_SINGULARITY` is set, my bad.